modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -271,7 +270,10 @@ const RichText = ({
 						<CKEditor5ClassicEditor
 							className="w-100"
 							config={{
-								...advancedClassicEditorConfig,
+								filebrowserImageBrowseUrl:
+									editorConfig.filebrowserImageBrowseUrl,
+								itemSelectorEventName: `${name}selectItem`,
```

Comment:
`itemSelectorEventName` is needed to wire event in item selector iframe JSP with editor. In CKE4 we did this wiring internally through editor ID, but editor ID can only be auto-generated ID in CKE5. See https://github.com/ckeditor/ckeditor5-react/issues/582